src/control-plane/backend/lambda/api/service/reporting.ts

```diff
@@ -1327,10 +1316,26 @@ export class ReportingService {
 
       //warm up redshift serverless
       if (latestPipeline.dataModeling?.redshift?.newServerless) {
-        const workgroupName = getStackOutputFromPipelineStatus(
+        const dataApiRole = getStackOutputFromPipelineStatus(
           latestPipeline.stackDetails ?? latestPipeline.status?.stackDetails,
-          PipelineStackType.DATA_MODELING_REDSHIFT,
-          OUTPUT_DATA_MODELING_REDSHIFT_SERVERLESS_WORKGROUP_NAME);
+          PipelineStackType.REPORTING,
+          OUTPUT_REPORTING_QUICKSIGHT_REDSHIFT_DATA_API_ROLE_ARN);
+        const redshiftEndpoint = getStackOutputFromPipelineStatus(
```

Comment:
Let's rely on the endpoint to check if it's serverless for the dashboard not the one in data modeling.